docs/ops/movement/Pad_12.md

```diff
@@ -0,0 +1,261 @@
+ [ 9,  9, 10, 11, 12, 12, 12, 12 ]]
+```
+* `pad_mode = "reflect"`:
+```
+OUTPUT =
+[[  2,  1,  2,  3,  4,  3,  2,  1 ]
+ [  6,  5,  6,  7,  8,  7,  6,  5 ]
+ [ 10,  9, 10, 11, 12, 11, 10,  9 ]
+ [  6,  5,  6,  7,  8,  7,  6,  5 ]
+ [  2,  1,  2,  3,  4,  3,  2,  1 ]]
+```
+* `pad_mode = "symmetric"`:
+```
+OUTPUT =
+[[ 1,  1,  2,  3,  4,  4,  3,  2 ]
+ [ 5,  5,  6,  7,  8,  8,  7,  6 ]
+ [ 9,  9, 10, 11, 12, 12, 11, 10 ]
+ [ 9,  9, 10, 11, 12, 12, 11, 10 ]
+ [ 5,  5,  6,  7,  8,  8,  7,  6 ]]
+```
+### Negative pads example:
+```
+pads_begin = [-1, -1]
+pads_end = [-1, -1]
+
+DATA =
+[[1,  2,  3,  4]
+ [5,  6,  7,  8]
+ [9, 10, 11, 12]]
+Shape(3, 4)
+```
+for all of the *pad_mode* attribute options:
+* `pad_mode = "constant"`
+* `pad_mode = "edge"`
+* `pad_mode = "reflect"`
+* `pad_mode = "symmetric"`
+```
+OUTPUT =
+[[ 6, 7 ]]
+Shape(1, 2)
+```
+### Mixed pads example:
+```
+pads_begin = [2, -1]
+pads_end = [-1, 3]
+
+DATA =
+[[1,  2,  3,  4]
+ [5,  6,  7,  8]
+ [9, 10, 11, 12]]
+Shape(3, 4)
+```
+* `pad_mode = "constant"`:
+```
+OUTPUT =
+[[0, 0, 0, 0, 0, 0],
+ [0, 0, 0, 0, 0, 0],
+ [2, 3, 4, 0, 0, 0],
+ [6, 7, 8, 0, 0, 0]]
+Shape(4, 6)
+```
+* `pad_mode = "edge"`:
+```
+OUTPUT Shape(4, 6) =
+[[2, 3, 4, 4, 4, 4],
+ [2, 3, 4, 4, 4, 4],
+ [2, 3, 4, 4, 4, 4],
+ [6, 7, 8, 8, 8, 8]]
+Shape(4, 6)
+```
+* `pad_mode = "reflect"`:
+```
+OUTPUT =
+[[10, 11, 12, 11, 10, 9],
+ [6,   7,  8,  7,  6, 5],
+ [2,   3,  4,  3,  2, 1],
+ [6,   7,  8,  7,  6, 5]]
+Shape(4, 6)
+```
+* `pad_mode = "symmetric"`:
+```
+OUTPUT =
+[[6, 7, 8, 8, 7, 6],
+ [2, 3, 4, 4, 3, 2],
+ [2, 3, 4, 4, 3, 2],
+ [6, 7, 8, 8, 7, 6]]
+Shape(4, 6)
+```
+
+
+**Attributes**
+
+* *pad_mode*
+
+  * **Description**: *pad_mode* specifies the method used to generate the padding values.
+  * **Range of values**: Name of the method in string format:
+    * `constant` - padded values are taken from the *pad_value* input. If the input is not provided, the padding elements are equal to zero.
+    * `edge` - padded values are copied from the respective edge of the input `data` tensor.
+    * `reflect` - padded values are a reflection of the input `data` tensor. Values on the edges are not duplicated, `pads_begin[D]` and `pads_end[D]` must be not greater than `data.shape[D] – 1` for any valid `D`.
+    * `symmetric` - padded values are symmetrically added from the input `data` tensor. This method is similar to the `reflect`, but values on edges are duplicated. Refer to the examples above for more details. `pads_begin[D]` and `pads_end[D]` must be not greater than `data.shape[D]` for any valid `D`.
+  * **Type**: `string`
+  * **Required**: *yes*
+
+**Inputs**
+
+* **1**: `data` tensor of arbitrary shape and type *T*. **Required.**
+
+* **2**: `pads_begin` 1D tensor of type *T_INT*. Number of elements matches the shape rank of *data* input. Specifies the number of padding elements to add at the beginning of each axis. Negative value means shrinking of the corresponding dimension's value. **Required.**
```

Comment:
may be instead of `shrinking` it is better to use `cropping`?

Reply:
Sounds good, updated